src/main/scala/managehelpcontentpublisher/S3.scala

```diff
@@ -59,4 +64,22 @@ object S3 {
 
   def putTopic(topic: PathAndContent): Either[Failure, PathAndContent] =
     put(s"${config.aws.topicsFolder}/${topic.path}.json", topic.content)
+
+  private def delete(key: String): Either[Failure, String] = {
+    val fullPath = s"s3://${config.aws.bucketName}/$key"
+    Try(
+      client.deleteObject(
+        DeleteObjectRequest
+          .builder()
+          .bucket(config.aws.bucketName)
+          .key(key)
+          .build()
+      )
+    ).toEither.left
```

Comment:
This is confusing formatting because first map is not immediately obvious to be a left map. Maybe break after `toEither` so that the left map is more explicit

```
toEither
  .left.map(...)
  .map(...)
```